manifests/12_prometheus_rule.yaml

```diff
@@ -13,47 +13,13 @@ metadata:
     role: alert-rules
 spec:
   groups:
-    - name: marketplace.community_operators.rules
+    - name: operator.marketplace.rules
       rules:
-        - alert: CommunityOperatorsCatalogError
+        - alert: OperatorHubSourceError
           annotations:
-            summary: Community-operators not ready for 10 minutes
-            description: Fires whenever the community-operators source is not ready for more than 10 mins.
-            message: Default OperatorHub source "community-operators" is in Non-Ready state for more than 10 mins.
-          expr: catalogsource_ready{name="community-operators",exported_namespace="openshift-marketplace"} == 0
-          for: 10m
-          labels:
-            severity: warning
-    - name: marketplace.certified_operators.rules
-      rules:
-        - alert: CertifiedOperatorsCatalogError
-          annotations:
-            summary: Certified-operators not ready for more than 10 minutes
-            description: Fires whenever the certified-operators source is not ready for more than 10 mins.
-            message: Default OperatorHub source "certified-operators" is in Non-Ready state for more than 10 mins.
-          expr: catalogsource_ready{name="certified-operators",exported_namespace="openshift-marketplace"} == 0
-          for: 10m
-          labels:
-            severity: warning
-    - name: marketplace.redhat_operators.rules
-      rules:
-        - alert: RedhatOperatorsCatalogError
-          annotations:
-            summary: Redhat-operators not ready for more than 10 minutes
-            description: Fires whenever the redhat-operators source is not ready for more than 10 mins.
-            message: Default OperatorHub source "redhat-operators" is in Non-Ready state for more than 10 mins.
-          expr: catalogsource_ready{name="redhat-operators",exported_namespace="openshift-marketplace"} == 0
-          for: 10m
-          labels:
-            severity: warning
-    - name: marketplace.redhat_marketplace.rules
-      rules:
-        - alert: RedhatMarketplaceCatalogError
-          annotations:
-            summary: Redhat-marketplace not ready for more than 10 minutes
-            description: Fires whenever the redhat-marketplace source is not ready for more than 10 mins.
-            message: Default OperatorHub source "redhat-marketplace" is in Non-Ready state for more than 10 mins.
-          expr: catalogsource_ready{name="redhat-marketplace",exported_namespace="openshift-marketplace"} == 0
+            summary: The {{ $labels.name }} source is in non-ready state for more than 10 minutes.
```

Comment:
From [the `summary` docs][1]:

> Use a static string or, if dynamic expansion is needed, aim for expanding into the same string for alerts that are typically grouped together into one notification. In that way, it can be used as a common “headline” for all alerts in the notification template.

So I'd probably leave `{{ $labels.name }}`.  But I haven't broken multiple catalogs and then gone to look at the UX; maybe it's fine with the expanded name.

[1]: https://github.com/monitoring-mixins/docs/pull/10/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R186-R189

Reply:
Is there often a need to distinguish between multiple instances of this alert?  I guess that depends on how often catalogs break from source-specific reasons vs. how often they break for all-source reasons?

Reply:
@wking I don't believe we have any instances where there are all-source reasons, unless for eg all the sources use images in a registry that's currently unreachable, or there's a bug that breaks all the CatalogSources. i.e there aren't any instance where all sources are down because of a single source, if that's what you were asking. So single source breaking is much more common place.